plugins/outputs/datadog/datadog.go

```diff
@@ -19,14 +19,17 @@ type Datadog struct {
 	Apikey  string
 	Timeout internal.Duration
 
-	apiUrl string
+	URL    string `toml:"url"`
 	client *http.Client
 }
 
 var sampleConfig = `
   ## Datadog API key
   apikey = "my-secret-key" # required.
 
+  # The base endpoint URL can optionally be specified but it defaults to:
+  url = "https://app.datadoghq.com/api/v1/series"
```

Comment:
Comment this line out since it won't usually need to be set:
```
  ## The base endpoint URL can optionally be specified but it defaults to:
  # url = "https://app.datadoghq.com/api/v1/series"
```

https://github.com/influxdata/telegraf/wiki/SampleConfig

Reply:
```suggestion
  url = "https://app.datadoghq.com/api/v1/series"
```
If the user is using an open source datadog-agent, this will be very useful to specify the url if they don't want to send the metrics to datadoghq.
For example:
`[centos@x.x.x.x telegraf.d]$ pwd
/var/lib/dcos/telegraf/telegraf.d
[centos@x.x.x.x telegraf.d]$ cat datadog.conf 
# Transmit all metrics to Datadog
[[outputs.datadog]]
  ## Datadog API key
  apikey = "redacted"
  ## Connection timeout
  # timeout = "5s"
  ## URL
  url = "http://custom_url:443/api/v1/metrics"`